miniproxy.gemspec

```diff
@@ -6,7 +6,12 @@ Gem::Specification.new do |s|
   s.name = 'miniproxy'
   s.version = MiniProxy::VERSION
   s.summary = 'Stub requests for browser tests'
-  s.authors = ["x"]
+  s.authors = ['The Conversation Dev Team']
+
+  s.summary = 'MiniProxy: stub requests for browser tests'
```

Comment:
This is pretty good. I was having a look on rubygems and the description doesn't need the name of the project because it's already displayed.

For example, this is what puffing billy has for its summary:

> Easy request stubs for browser tests.

and description:

> A stubbing proxy server for ruby. Connect it to your browser in integration tests to fake interactions with remote HTTP(S) servers.

Maybe we could have something like:

> Easily stub HTTP requests for your browser tests.

and description:

> A tool which allows you to easily stub requests to external sites in your browser tests.

Reply:
🏆 descriptions! Updating!